docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -9,24 +9,50 @@ description: Managing your review workflows and stages in Strapi.
 Review Workflows is an <EnterpriseBadge />-only feature.
```

Comment:
I think I'd delete this note, as the Enterprise badge is already displayed in the title. It looks a bit weird to have the note as well, which we also never used elsewhere (including in Single Sign-On which is right after this page and also an Enterprise feature).

Reply:
This has been added outside of this PR https://github.com/strapi/documentation/commit/b9cdfe7213ebf7cc4d649b39645669940449c7b8

I am fine with removing it. @pwizla ?

Reply:
@meganelacheny I added this block last week. I agree it's not really beautiful, but I added it here and in some other EE-only pieces of documentation when handling users' feedback, to make it super explicit because some users were surprised the features were not available in their setup. If you have any idea about how we can make it better, please let me know 🤗

Reply:
Oh I see! Sorry I didn't see it earlier. Well in that case I guess we can leave it like that for now so we don't block this pull request, and we can discuss together how we could improve this 🤗